### DISCUSSION TITLE
Issues encountered when manually installing the tribe project

### DISCUSSION THREAD

**Discussion by purin-blog** [StreetLamb/tribe#131] [Q&A]
I manually run the node:20 container to download the docker image. Then I cloned the tribe project from github. After the cloning is complete, I typed the cd tribe command to switch to the Tribe project directory. I connected to the Tribe container through the remote development extension of VSCode. When I navigated to the tribe folder inside the container, I typed cp .env.example .env to copy the contents of the .env file I created to /tribe/ in the container, replacing the unmodified .env file. Finally, I typed npm install && npm start to start the Tribe application.
I got the following output:
root@7f220b3a7114:/tribe# npm install && npm start
npm error code ENOENT
npm error syscall open
npm error path /tribe/package.json
npm error errno -2
npm error enoent Could not read package.json: Error: ENOENT: no such file or directory, open '/tribe/package.json'
npm error enoent This is related to npm not being able to find a file.
npm error enoent
npm error A complete log of this run can be found in: /root/.npm/_logs/2024-08-26T08_55_52_910Z-debug-0.log
At the same time, I did not find the accessible URL http://traefik.localhost/ in the tribe docker container

**Comment by StreetLamb** [comment]
Hi @purin-blog, I dont think your deployment steps are correct. Does the [local deployment steps](https://github.com/StreetLamb/tribe/blob/master/local-deployment.md) help?

**Reply by purin-blog** [reply]
I followed the local deployment steps to configure the environment variables. I used a proxy, and the docker-compose -f docker-compose.yml -f docker-compose.local.yml up command did not work for me.

**Reply by StreetLamb** [reply]
I see, likely your proxy is preventing you from downloading the required images from the docker hub. Can you post the error logs after running `docker compose -f docker-compose.yml -f docker-compose.local.yml up`?

**Comment by tt-a1i** [comment]
The ESLint `camelcase` rule is being implemented in PR #16908. Once merged, you'll be able to enable it with:

```json
{
  "rules": {
    "eslint/camelcase": "error"
  }
}
```

The rule supports the same options as ESLint: `properties`, `ignoreDestructuring`, `ignoreImports`, and `allow` patterns.
